doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1359 @@
+* Improved time performance of firmware update by usage of best RRAM write operation buffering on nRF54L SoCs.
+* Introduced improved swap algorithm: swap-using-offset, see ``CONFIG_BOOT_SWAP_USING_OFFSET`` MCUboot Kconfig option.
+* Image compression support has been brought to production quality.
+
+Zephyr
+======
+
+.. NOTE TO MAINTAINERS: All the Zephyr commits in the below git commands must be handled specially after each upmerge and each nRF Connect SDK release.
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``fdeb7350171279d4637c536fcceaad3fbb775392``, with some |NCS| specific additions.
+
+For the list of upstream Zephyr commits (not including cherry-picked commits) incorporated into nRF Connect SDK since the most recent release, run the following command from the :file:`ncs/zephyr` repository (after running ``west update``):
+
+.. code-block:: none
+
+   git log --oneline fdeb735017 ^beb733919d
+
+For the list of |NCS| specific commits, including commits cherry-picked from upstream, run:
+
+.. code-block:: none
+
+   git log --oneline manifest-rev ^fdeb735017
+
+The current |NCS| main branch is based on revision ``fdeb735017`` of Zephyr.
+
+.. note::
+   For possible breaking changes and changes between the latest Zephyr release and the current Zephyr version, refer to the :ref:`Zephyr release notes <zephyr_release_notes>`.
+
+Additions specific to |NCS|
+---------------------------
+
+The new ZMS Settings backend is not backward compatible with the old version.
+
+Using the new backend you can now enable some performance optimizations using the following Kconfig options:
+
+* :kconfig:option:`CONFIG_SETTINGS_ZMS_LL_CACHE` - Used for caching the linked list nodes related to Settings Key/Value entries.
+* :kconfig:option:`CONFIG_SETTINGS_ZMS_LL_CACHE_SIZE` - The size of the linked list cache (each entry occupies 8 Bytes of RAM).
+* :kconfig:option:`CONFIG_SETTINGS_ZMS_NO_LL_DELETE` - Disables deleting the linked list nodes when deleting a Settings Key.
+  Use this option only when the application is always using the same Settings Keys.
+  When the application uses random Keys, enabling this option could lead to incrementing the linked list nodes without corresponding Keys and cause excessive delays to loading of the Keys.
+  Use this option only to accelerate the delete operation for a fixed set of Settings elements.
+* :kconfig:option:`CONFIG_SETTINGS_ZMS_LOAD_SUBTREE_PATH` - Loads first the subtree path passed in the argument, then continue to load all the Keys in the same subtree if the handler returns a zero value.
+* Using :c:func:`settings_load_one` function to retrieve the value of a known Key is recommended with ZMS as it improves the read performance a lot.
+* To get the value length of a Settings entry, it is recommended to use the :c:func:`settings_get_val_len` function.
+
+zcbor
+=====
```

Comment:
@oyvindronningstad, anything to add here?

Reply:
No nothing new this release.

Reply:
Removed the section